portfolio.html

```diff
@@ -0,0 +1,603 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="portfolio.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <nav>
+        <div class="lower-bar">
+          <div class="haftamu">Haftamu</div>
+
+          <img
+            src="bar/Enabled.png"
+            class="allign"
+            alt="hamburger menu symbol"
+          />
+          <p class="close-menu">&times;</p>
+        </div>
+        <div class="contacts-menu">
+          <ul>
+            <li>
+              <a href="#section11" class="navlinks">Portfolio</a>
+              <hr />
+            </li>
+            <li>
+              <a href="#section33" class="navlinks">About</a>
+              <hr />
+            </li>
+            <li>
+              <a href="#section44" class="navlinks">Contact</a>
+              <hr />
+            </li>
+          </ul>
+        </div>
```

Comment:
### Your Navbar
![menu-icon (another copy)](https://user-images.githubusercontent.com/93039359/179872851-b1a9fc04-81eb-4fd0-a9f2-9b4c6a184e0e.png)

- [ ] Your logo should navigate top to your portfolio and your Portfolio link should navigate to the projects section of your portfolio. :+1: Kindly fix it :blush: